packages/app/src/messages/components/message-bubble/message-bubble.component.tsx

```diff
@@ -89,7 +89,7 @@ const MessageBubble: FunctionComponent<MessageBubbleProps> = ({
           data-testid={MessageBubbleTestIds.Container}
           interlocutor={interlocutor}
         >
-          {isDropdownShouldVisible() && (
+          {isDropdownVisible() && !messageSending && (
```

Comment:
Maybe we could add `!messageSending` condition to `isDropdownVisible` function? By that we would have validation in one place.

Reply:
Good point :) [8c78439](https://github.com/mudita/mudita-center/pull/1010/commits/8c7843918dd4aac4bf0fd07faac22b21a73bd801)